src/shared/components/FilterList/InjectionOption.tsx

```diff
@@ -48,6 +50,21 @@ const FluxInjectionOption: FC<Props> = ({
   const handleClick = () => {
     onClick(option)
   }
+  const fullClick = () => {
+    if (!isFlagEnabled('fluxEditorButtonStyling')) {
+      return
+    }
+    onClick(option)
+  }
+
+  const classer = [
+    ['flux-toolbar--list-item', true],
+    ['flux-toolbar--function', true],
+    ['flux-toolbar--new-style', isFlagEnabled('fluxEditorButtonStyling')],
+  ]
+    .filter(c => c[1])
+    .map(c => c[0])
+    .join(' ')
```

Comment:
but like ... why have to reason about it and not just:

`flux-toolbar--list-item flux-toolbar--function ${isFlagEnabled ? 'flux-toolbar--new-style}`

Reply:
Oh this is a copy pasta job i started inserting a year ago instead of using classer. I like the readability of the format (makes more sense as complexity grows), but it felt weird maintaining a dependency for it